pkg/framework/plugins/nodeutilization/highnodeutilization.go

```diff
@@ -71,7 +72,14 @@ func (h *HighNodeUtilization) Name() string {
 }
 
 // Balance extension point implementation for the plugin
-func (h *HighNodeUtilization) Balance(ctx context.Context, nodes []*v1.Node) *framework.Status {
+func (h *HighNodeUtilization) Balance(ctx context.Context, nodes []*v1.Node) (status *framework.Status) {
+	ctx, span, spanCloser := tracing.StartSpan(ctx, tracing.BalanceOperation, h.Name())
```

Comment:
I'm debating whether to have this one level up, so it doesn't have to be repeated in every plugin and if we add a new one we won't forget to add this.

If it's moved up, does it mean that we lose some context?

Reply:
That was the original idea. But @damemi  suggested that it would be fine to let the individual plugins handle this on their own. I am open to moving this to a different location if that is the need.

Reply:
We could just do this trace around the calls to the plugins, rather than in the plugin itself (I think that's what you mean by `have this one level up` @a7i right?) I didn't mean to imply that each plugin *has* to handle this on their own, sorry if that was confusing.

That would make this PR a lot smaller, much easier to review for a first pass. And we can always add spans inside the plugins later if we want.